opennms-doc/guide-admin/src/asciidoc/text/user-management/security-roles.adoc

```diff
@@ -7,12 +7,7 @@
 
 A _Security Roles_ is a set of permissions and can be assigned to an _User_.
 They regulate access to the Web User Interface and the _ReST API_ to exchange monitoring and inventory information.
-ifndef::opennms-prime[]
-In case of a distributed installation, the _Minion_ or _Remote Poller_ instances interact with _{opennms-product-name}_ and require specific permissions which are defined in the _Security Roles_ _ROLE_MINION_ and _ROLE_REMOTING_.
-endif::opennms-prime[]
-ifdef::opennms-prime[]
-In case of a distributed installation, the _Remote Poller_ instances interact with _{opennms-product-name}_ and require specific permissions which are defined in the _Security Role_ _ROLE_REMOTING_.
-endif::opennms-prime[]
+In case of a distributed installation, the _Minion_ instances interact with _{opennms-product-name}_ and require specific permissions which are defined in the _Security Roles_ _ROLE_MINION_.
```

Comment:
@indigo423 can you confirm that this switch is no longer needed, as the minion is available in latest prime?